
diff --git a/src/js/src/dropdown.js b/src/js/src/dropdown.js
index 51a80b13f..705807856 100644
--- a/src/js/src/dropdown.js
+++ b/src/js/src/dropdown.js
@@ -5,36 +5,6 @@ Core dropdowns
  */
 let dropdownTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="dropdown"]'));
 dropdownTriggerList.map(function (dropdownTriggerEl) {
+	console.log(dropdownTriggerEl.dataset);
 	return new Dropdown(dropdownTriggerEl);
-});
-
-
-/*
-Nested dropdowns
- */
-const selectors = '.dropdown, .dropup, .dropend, .dropstart',
-	dropdowns = document.querySelectorAll(selectors);
-
-let currentTarget = undefined;
-
-dropdowns.forEach(dropdown => {
-	dropdown.addEventListener('mousedown', (e) => {
-		e.stopPropagation();
-
-		if (e.target.dataset.bsToggle && e.target.dataset.bsToggle === 'dropdown') {
-			currentTarget = e.currentTarget;
-		}
-	});
-
-	dropdown.addEventListener('hide.bs.dropdown', (e) => {
-		e.stopPropagation();
-
-		const parent = currentTarget ? currentTarget.parentElement.closest(selectors) : undefined;
-
-		if (parent && parent === dropdown) {
-			e.preventDefault();
-		}
-
-		currentTarget = undefined;
-	});
 });
\ No newline at end of file

diff --git a/src/js/src/dropdown.js b/src/js/src/dropdown.js
index 705807856..35979fefb 100644
--- a/src/js/src/dropdown.js
+++ b/src/js/src/dropdown.js
@@ -5,6 +5,5 @@ Core dropdowns
  */
 let dropdownTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="dropdown"]'));
 dropdownTriggerList.map(function (dropdownTriggerEl) {
-	console.log(dropdownTriggerEl.dataset);
 	return new Dropdown(dropdownTriggerEl);
 });
\ No newline at end of file
